crates/storage/src/lib.rs

```diff
@@ -7,7 +7,7 @@ mod prelude;
 
 mod bloom;
 pub use bloom::BLOCK_RANGE_LEN;
-mod connection;
+pub mod connection;
```

Comment:
Can we avoid making the `connection` module public? We already expose all the contents via `pub use connection::*` from the crate root so this seems redundant?